### ISSUE TITLE
source code out of sync between github and cinnamon website

### ISSUE THREAD

**Issue by rickyzhang82** [fthuin/yahoofinance-cinnamon-desklet#3]
I found the source code between https://cinnamon-spices.linuxmint.com/desklets/view/23 and github are not in sync. Could you please tell me which version is up-to-date?

**Comment by fthuin** [comment]
Hello,
This repository contains the code merged after a pull request (see [https://github.com/fthuin/yahoofinance-cinnamon-desklet/pull/2](https://github.com/fthuin/yahoofinance-cinnamon-desklet/pull/2)). It contains a major refactoring of the code to be object-oriented and to handle errors.

The source code on cinnamon-spices contains the old version (non-refactored) but with other improvements (errors handling).

tl;dr: If you want to contribute, the best is to work on the source code in this repository.
